spec/requests/invites_controller_spec.rb

```diff
@@ -135,7 +135,7 @@
     end
   end
 
-  context '#create_invite_link' do
+  context '#create_single_use_invite_link' do
```

Comment:
`#` is meant to represent an instance method so I would probably rearrange this to 

```
describe "#create_invite_link" do
  describe "single use invite link" do
    ...
  end 
  describe "multiple use invite link" do
    ...
  end
end
```